community/runc/APKBUILD

```diff
@@ -0,0 +1,45 @@
+# Contributor: Jake Buchholz <tomalok@gmail.com>
+# Maintainer: Jake Buchholz <tomalok@gmail.com>
+
+pkgname=runc
+
+# NOTE: using explicit post-1.0.0_rc6 commit, for containerd-1.2.1,
+# see https://github.com/containerd/containerd/blob/v1.2.1/RUNC.md
+# and https://github.com/containerd/containerd/blob/v1.2.1/vendor.conf
+_commit=96ec2177ae841256168fcf76954f7177af9446eb
+
+pkgver=1.0.0_rc6
+pkgrel=0
+pkgdesc="CLI tool for spawning and running containers according to the OCI specification"
+url="https://www.opencontainers.org"
+arch="all"
+license="Apache-2.0"
+depends="libseccomp"
```

Comment:
I don't think the explicit depend of libseccomp is needed. It is automatically detected:
````
>>> runc*: Tracing dependencies...
	libseccomp
	so:libc.musl-x86_64.so.1
	so:libseccomp.so.2
````

Reply:
i'll keep that in mind for the next update, thanks!